src/components/guideLineLayer.ts

```diff
@@ -0,0 +1,154 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Components {
+  export class GuideLineLayer<D> extends Component {
+    public static ORIENTATION_VERTICAL = "vertical";
+    public static ORIENTATION_HORIZONTAL = "horizontal";
+
+    private _orientation: string;
+    private _value: D;
+    private _scale: QuantitativeScale<D>;
+    private _pixelPosition: number;
+    private _scaleUpdateCallback = () => {
+      this._updatePixelPosition();
+      this.render();
+    };
```

Comment:
I'm personally surprised they allow code blocks like this in instance variable initialization.

So far I don't have an opinion on this